lib/config/definitions.ts

```diff
@@ -593,26 +613,26 @@ const options: RenovateOptions[] = [
     description: 'Version scheme to use for filtering and comparisons',
     type: 'string',
     allowedValues: [
-      'cargo',
-      'composer',
-      'docker',
-      'git',
-      'hashicorp',
-      'hex',
-      'ivy',
-      'loose',
-      'maven',
-      'node',
-      'npm',
-      'nuget',
-      'pep440',
-      'poetry',
-      'regex',
-      'ruby',
-      'semver',
-      'swift',
+      VERSION_SCHEME_CARGO,
+      VERSION_SCHEME_COMPOSER,
+      VERSION_SCHEME_DOCKER,
+      VERSION_SCHEME_GIT,
+      VERSION_SCHEME_HSCORP,
+      VERSION_SCHEME_HEX,
+      VERSION_SCHEME_IVY,
+      VERSION_SCHEME_LOOSE,
+      VERSION_SCHEME_MAVEN,
+      VERSION_SCHEME_NODE,
+      VERSION_SCHEME_NPM,
+      VERSION_SCHEME_NUGET,
+      VERSION_SCHEME_PEP440,
+      VERSION_SCHEME_POETRY,
+      VERSION_SCHEME_REGEX,
+      VERSION_SCHEME_RUBY,
+      VERSION_SCHEME_SEMVER,
+      VERSION_SCHEME_SWIFT,
     ],
```

Comment:
We should think about (later) if we can automatically import this from the constants

Reply:
I already have an idea about that, but it was out of scope for this PR, so skipped that.